### ISSUE TITLE
Able to wear two backpack items at once.

### ISSUE THREAD

**Issue by Cameron653** [VOREStation/VOREStation#394]
#### Brief description of the issue

You're able to wear two backpack items at once.
#### What you expected to happen

Only be able to wear one BP item at once.
#### What actually happened

Be able to wear two.
#### Steps to reproduce

**1.** Get an AMI control module/any rigsuit module
**2.** Put rigsuit module in hand, dragdrop backpack to hand that is holding the module
**3.** Turn on hotkey mode
**4.** Press E with the rigsuit module in hand, wait a bit, and you'll be able to wear two backpack items at once.
**5.** Have this happen: http://puu.sh/pN2yX/ec879c0d2f.png
#### Additional info:
- **Server Revision**: release - 2016-06-15
- **Anything else you may wish to add** 

I wish this was a feature.

**Comment by Screemonster** [comment]
This has happened to Scree. His jumpsuit starts in his back slot along with his satchel, and hitting ctrl-e with my jumpsuit slot empty puts it in my back slot again.

Last shift, a Teshari was able to climb on my back when I had my backpack on.

Notably they were NOT able to do so if I took my satchel off at the start of the shift and put it back on.

**Comment by SpadesNeil** [comment]
Still a bug?

**Comment by Screemonster** [comment]
I _think_ it was fixed for the fluff jumpsuit issue as the check for "can this character wear this thing" was overriding the "it it possible to wear this thing in this slot at all" check.

The bug DOES seem to still happen with rigsuits if you follow the (somewhat esoteric) steps to reproduce there. I found that my backpack sprite stayed stuck in my hand (and the contents could still be accessed) but I could still pick stuff up in that hand. Dragging the thing onto a rigsuit seems to stop it from "occupying" whatever slot it's in.
